components/Login/style.module.css

```diff
@@ -0,0 +1,14 @@
+.login-form-button {
+  width: 100%;
+}
+
+.double-column {
```

Comment:
We mostly can write single word classes because these modules are usually relatively short, like `.columns` could be a good way to go.

Reply:
I know, but sometimes a self-explanatory name makes more sense...

But because `className={styles["double-column"]}` is also terrible, I will oblige

Reply:
I think maybe when we really really need to use double words we could try camelCase (I hated it in css class names) but at least we don't lose the linter and editor features.

Reply:
Actually, I just tested it and the linter is not testing for non-existing class names, we should check that ¯\_(ツ)_/¯